go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -180,44 +191,74 @@ func (ef *egressFirewall) addACLToJoinSwitch(joinSwitches []string, hashedAddres
 			action = "drop"
 		}
 
-		ipAddress, _, err := net.ParseCIDR(rule.to.cidrSelector)
-		if err != nil {
-			return fmt.Errorf("error rule.to.cidrSelector %s is not a valid CIDR (%+v)", rule.to.cidrSelector, err)
-		}
-		if !utilnet.IsIPv6(ipAddress) {
-			match = fmt.Sprintf("match=\"ip4.dst == %s && ip4.src == $%s\"", rule.to.cidrSelector, hashedAddressSetNameIPv4)
+		if rule.to.cidrSelector != "" {
```

Comment:
Suggest you factor out match generation in to a separate, referentially transparent function for ease of testing.

Reply:
done